### ISSUE TITLE
Windows - Hyper terminates in 5h intervals

### ISSUE THREAD

**Issue by peterdanis** [zeit/hyper#2376]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Microsoft Windows [Version 6.1.7601]
- **Hyper.app version**: 1.4.8 and 2.0.4
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/peterdanis/52d74e683cdb65dd026730b6f3a4c330
- **Relevant information from devtools**: N/A
- **The issue is reproducible in vanilla Hyper.app**: I didn't try yet

## Issue
Edited, see  https://github.com/zeit/hyper/issues/2376#issuecomment-341637586
~Sometimes when I have Hyper app running, I lock (Win + L) my Windows machine + let the screen turn off (after 15 min of inactivity), and after returning to PC the Hyper app is not running (without any crash report on the screen). I have searched event log, appdata and temp folders but did not find any kind of Hyper app logs. May I kindly ask you for any ideas how to debug my issue?~

**Comment by albinekb** [comment]
Hi! Thanks for reporting 👍 

There are several reports on similar things, but I don't know if it's connected to the issue you're describing:

https://github.com/zeit/hyper/issues/2168
https://github.com/zeit/hyper/issues/1952

This is probably different though... And if you can't find any crash reports, that's really weird..

**Comment by albinekb** [comment]
Could it maybe be related to the display saving logic, there are some problems with that: https://github.com/zeit/hyper/issues/1869

Maybe a long shot too..

**Comment by peterdanis** [comment]
I have found out that the hyper app termination is not related to screen or display. It terminates every 5 hours for me. There is no crash report or anything, it just terminates silently.

_Left section of screenshot:_ Power on date and time. Hyper is starting automatically with Windows (via shortcut in Startup folder).
_Right section of screenshot:_ I run Powershell as shell in Hyper, it writes an event to event log when it transitions from "available" to "stopped" state. 

![image](https://user-images.githubusercontent.com/26599181/32363767-366a95f4-c071-11e7-99cf-92504a49d903.png)

I think `updatePlugins` is failing here. I did not find anything other running in 5h interval.

In `app/plugins.js`:
```
// otherwise update plugins every 5 hours
setInterval(updatePlugins, ms('5h'));
